server/schedule/filters.go

```diff
@@ -412,3 +412,60 @@ func (f StoreStateFilter) filterMoveRegion(opt Options, store *core.StoreInfo) b
 	}
 	return false
 }
+
+// BlacklistType the type of BlackListStore Filter.
+type BlacklistType int
+
+// some flags about blacklist type.
+const (
+	SignSource BlacklistType = iota + 1
```

Comment:
We need a better name or some comments for it.

Reply:
What does `Sign` mean?